pom.xml

```diff
@@ -29,26 +29,26 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.java.source>1.8</project.java.source>
+    <project.java.source>17</project.java.source>
     <!-- dependency versions -->
-    <assertj.version>3.11.1</assertj.version>
-    <bagit.version>5.1.1</bagit.version>
-    <commons-io.version>2.6</commons-io.version>
-    <commons-compress.version>1.20</commons-compress.version>
-    <junit.version>4.13.1</junit.version>
-    <logback.version>1.1.2</logback.version>
-    <slf4j.version>1.7.10</slf4j.version>
-    <tika.version>1.23</tika.version>
-    <yaml-beans.version>1.13</yaml-beans.version>
+    <assertj.version>3.26.3</assertj.version>
+    <bagit.version>5.2.0</bagit.version>
+    <commons-io.version>2.16.1</commons-io.version>
+    <commons-compress.version>1.27.1</commons-compress.version>
+    <junit.version>5.11.0</junit.version>
+    <logback.version>1.5.8</logback.version>
+    <slf4j.version>2.0.16</slf4j.version>
+    <tika.version>2.9.2</tika.version>
+    <yaml-beans.version>1.17</yaml-beans.version>
     <!-- plugin versions -->
     <codestyle.version>1.0.1</codestyle.version>
```

Comment:
I noticed the other duracloud projects are using `1.1.0` here. Not sure if we want to update it or not in this PR.